components/concept/engine/src/main/java/mozilla/components/concept/engine/media/Media.kt

```diff
@@ -178,7 +178,9 @@ abstract class Media(
      * @property duration Indicates the duration of the media in seconds.
      */
     data class Metadata(
-        val duration: Double = -1.0
+        val duration: Double = -1.0,
+        val height: Long = 0L,
+        val width: Long = 0L
     )
```

Comment:
In #4287 I am currently refactoring media / feature-media to not use `browser-session` anymore (and `browser-state` instead). Let's try to not break each other's code. :)

Maybe after the refactoring we can also make it easier to choose the right orientation - since with `browser-state` we do not depend on callback order that much. But that's something I'd have to check after I have it working. :)